internal/docs/release.md

```diff
@@ -0,0 +1,31 @@
+# Releases
+
+We aim to do a release after each Sprint (roughly, every 3 weeks).
+
+To do a release:
+
+-   Identify the [commit](https://github.com/google/go-cloud/commits/master) you
```

Comment:
I've found the slightly easier view is https://github.com/google/go-cloud/compare and then plugging in the latest version tag as the base. You can find the latest release by visiting the magic URL https://github.com/google/go-cloud/releases/latest, but that might be too much magic. 😀

Reply:
Added a link to that.